verify_rpms/rpm_verifier.py

```diff
@@ -112,16 +127,32 @@ def generate_output(processed_image: ProcessedImage) -> tuple[bool, str]:
     return failure, output.lstrip()
 
 
+def generate_results(processed_image: ProcessedImage) -> dict[str, Any]:
+    """
+    Generate the results that should be added to the results of the task
+    :param processed_image: ProcessedImage that contains all the rpms information
+    :returns: dictionary with results for the image
+    """
+    keys: dict[str, Any] = {}
```

Comment:
I think we should call that dict `out` or `results` or something like that. `keys` is confusing as it's not holding just the keys.